sample-apps/react-native/dogfood/App.tsx

```diff
@@ -77,12 +77,14 @@ const StackNavigator = () => {
   useEffect(() => {
     const subscription = prontoCallId$.subscribe((prontoCallId) => {
       if (prontoCallId) {
-        setState({ appMode: 'Meeting' });
+        if (username && userImageUrl) {
```

Comment:
its better to move this if three lines before.. so that we don't subscribe if the user stuff is not available

Reply:
Done